### PR TITLE
docs: demos subnav tab

### LINKED ISSUE #1452: [docs] move playground to code tab
_This may be a duplicate issue or one of many RFEs or Bugs out there. If so, let's make this an Epic to track everything._

Our current live demo needs some improvements, including load time.

Related issues:

- [ ] View full demo link — [1451](https://github.com/RedHat-UX/red-hat-design-system/issues/1451)
- [ ] Relocate Demo as own "tab" (i.e. playground w/ knobs) — [1453](https://github.com/RedHat-UX/red-hat-design-system/issues/1453)
- [ ] Bug: `<rh-playground>` external content included in demo editor — [1255](https://github.com/RedHat-UX/red-hat-design-system/issues/1255)

### Current look
image

### LINKED ISSUE #1453: [docs] Relocate Demo as own subnav tab
Demos may deserve their own tab in the Elements' Subnav.

### PR DESCRIPTION
**Pull request by bennypowers**
Closes #1453 
Closes #1452 

## What I did

1. remove demos from overview pages
2. add a /demos/ subnav tab to elements
## Testing Instructions
- [Call to Action demos](https://deploy-preview-1648--red-hat-design-system.netlify.app/elements/call-to-action/demos/) and others

### CODE REVIEW

File: docs/_includes/partials/component/back-to-top.njk [hunk 1]
```diff
@@ -1,45 +1,2 @@
...
-  #back-to-top-q3x4zkd {
-    display: none;
-    pointer-events: none;
-    position: sticky;
-    justify-content: flex-end;
-    width: 100%;
-    bottom: 32px;
-    margin-block-start: -64px;
-    padding-inline-end: 32px;
-  }
```
Comment (zeroedin) [inline docs/_includes/partials/component/back-to-top.njk:13]: we should reapply these custom styles so the back to top displays as you scroll rather then just when you reach the bottom of the page.  

```
rh-back-to-top {
  display: flex;
  pointer-events: all;
  position: sticky;
  justify-content: flex-end;
  width: 100%;
  bottom: 32px;
  margin-block-start: -64px;
  padding-inline-end: 32px;
 }
```

The above snippet seems to do it.

Comment (bennypowers) [reply docs/_includes/partials/component/back-to-top.njk:13]: the element doesn't do that on it's own?

Comment (zeroedin) [reply docs/_includes/partials/component/back-to-top.njk:13]: I would call the above specific to the website.  Depends on how you want to position it on the page.  The component will hide and show based on scroll but doesn't apply custom positioning.  `rh-back-to-top` is pretty much a copy / paste duplicate from `pf-back-to-top`.  I think we have room for improvement there.

File: docs/_includes/layouts/pages/elements.njk [hunk 2]
```diff
@@ -0,0 +1,82 @@
+---
```
Comment (zeroedin) [inline docs/_includes/layouts/pages/elements.njk:1]: Good call moving this into layouts from elements/elements.html

File: docs/_includes/partials/component/sidenav.njk [hunk 3]
```diff
@@ -2,14 +2,14 @@
   <ul>
     <li>
       <uxdot-sidenav-item {{ 'active' if page.url ===  '/'}}>
-        <a href="{{ '/' | url }}">Home</a>
```
Comment (zeroedin) [inline docs/_includes/partials/component/sidenav.njk:5]: Most likely is a good idea to go back and remove all these.  I should have done so prior to the docs PR merge.  My thought at the time was about the `/v2/` etc directories and the `pathPrefix`.  Was going to try it and the `<base>` plugin and it became an incomplete thought.  

> If you need to deploy your site to a subfolder
https://www.11ty.dev/docs/filters/url/

I'll split this out into an separate issue and take care of all the removals.

Comment (zeroedin) [reply docs/_includes/partials/component/sidenav.njk:5]: https://github.com/RedHat-UX/red-hat-design-system/issues/1677

**Review by zeroedin — APPROVED** [review]
Live gaming testing mechanics

**Pull request merged**